Makefile

```diff
@@ -19,62 +19,63 @@ clean:
 	find . -name "__pycache__" -exec rm -rf {} +
 	find . -type d -empty -delete
 
-cov:
-	$(PYTHON) -m coverage report
+cov: dev
+	$(UV) run coverage report
 
-cov-html:
-	$(PYTHON) -m coverage html
+cov-html: dev
+	$(UV) run coverage html
 	open htmlcov/index.html
 
-cov-xml:
-	$(PYTHON) -m coverage xml
-
-deps: ensure-uv
-	$(UV) pip install --upgrade pip setuptools wheel -r requirements.txt -r requirements-dev.txt
+cov-xml: dev
+	$(UV) run coverage xml
 
 dev: ensure-uv
 	$(UV) pip install -e .
 
-docs:
+docs: ensure-uv
 	$(MAKE) -C ./docs
 
-ensure-uv:
+$(VIRTUAL_ENV):
+	$(UV) venv $(VIRTUAL_ENV)
+_ensure-uv-venv: _ensure-uv-cmd $(VIRTUAL_ENV)
+_ensure-uv-cmd:
 	@if ! command -v $(UV) >/dev/null; then \
```

Comment:
Could this be a single command?

Reply:
Yes.

Converting it to a `$(MAKE) $(VIRTUAL_ENV)` call after `uv` has been installed